test/runtests.jl

```diff
@@ -2,8 +2,11 @@ using Compat.Dates
 using Compat.Test
 using PowerSystemsUnits
 
-using Unitful: @u_str
+Unitful.register(PowerSystemsUnits)
```

Comment:
Unitful is not yet defined at this point; this should come after `using Unitful`.

Reply:
oops! Should be fixed now.